autogluon/training/docker/0.8/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,68 @@
+ARG PYTHON_VERSION=3.9.16
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.13.1-cpu-py39-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
+# ENV PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
```

Comment:
should we completely remove this?

Reply:
Yea, I just tried removing it and did a smoke test for timeseries, looks good

Reply:
We shall remove this as it is resolved as part of CUDA 1.13.1 upgrade